scripts/rollup/build.js

```diff
@@ -194,13 +194,21 @@ function getFilename(name, hasteName, bundleType) {
   }
 }
 
-function uglifyConfig({
-  mangle,
-  manglePropertiesOnProd,
-  preserveVersionHeader,
-  removeComments,
-  headerSanityCheck,
-}) {
+/**
+ * @param configs {
```

Comment:
> Or it doesn't run in some Node version?

This was the reason.

> I'm just hesitant to add comments that aren't checked by a tool, and pretty much repeat the code.

Agreed that comments are kind of no-value-add here. They're a bit of extra maintenance burden but otherwise you might as well just scan below.